New docs structure (#56)
opengisch/QField-docs: en/field-work/digitize.rst

Proceed with adding points to form you line or polygon by clicking the **Plus (+) button** each time you want to add a new node.

Click the **Minus (-) button** to remove the last added node.

Click the **Close (X) button** to cancel the feature creation completely.

When you have added a least 2 nodes for a line or 3 nodes for a polygon, a **Save button** will appear. Click on it to finish your sketch.

.. image:: ../images/user-guide_digit3.png

Attribute form
--------------

After digitizing a geometry, the attribute form will be displayed if it is not suppressed
and the user will be asked to enter the attributes for the new feature.
The form which appears allows entering attribute values for the new feature. The checkboxes
at the right of every attribute allow for remembering each attribute individually.
The next time you will add a feature on the same layer, these attributes will be already pre-filled.

.. image:: ../images/user-guide_digit4.png

Delete features
===============

Deleting features is only possible in digitize mode.

> To delete a feature, identify it first and tap the trash icon.

Reviewer: ```suggestion
To delete a feature, identify it first by a short tap on it. Then select it with a long press
in the identification list, select further features to delete if you want and click the
trash button to confirm the deletion.
```

Edit features
=============

To edit, tap on **Menu button -> pencil.**

You can edit an existing feature by moving, adding or deleting the vertex and also spliting. 

Vertex tool
-----------

> Below, you see how the vertex tool works.

Reviewer: ```suggestion
It is possible to move and delete vertices of geometries and add new ones.
```

.. container:: clearer text-left

  .. figure:: ../images/edit_geom_feature.gif 
     :width: 300px
     :alt: edit_geom_feature

Split tool
----------

> Below, you see how the split tool works.

Reviewer: ```suggestion
It is possible to split an existing geometry by drawing a line where it should be split.
```
